README.md

```diff
@@ -185,79 +185,105 @@ import (
+    ImageTag:  "latest",
+	VaultLicense: licenseString, // not a path, the actual license bytes
+  }
+  cluster := docker.NewTestDockerCluster(t, opts)
+  defer cluster.Cleanup()
 }
 ```
 
-Here is a more realistic example.  This test should be run with VAULT_BINARY pointing
-at a local Linux vault binary on disk that you wish to run inside the docker containers. 
-If it's an Enterprise binary, you'll also need to set VAULT_LICENSE_CI, or the
-DockerClusterOptions field VaultLicnese. 
+Here is a more realistic example of how we use it in practice.  DefaultOptions uses 
+`hashicorp/vault`:`latest` as the repo and tag, but it also looks at the environment
+variable VAULT_BINARY. If populated, it will copy the local file referenced by
+VAULT_BINARY into the container. This is useful when there is no published image
+containing the vault binary you want to test.
+
+Instead of setting the VaultLicense option, you can set the VAULT_LICENSE_CI environment
+variable, which is better than committing a license to version control.
 
 Optionally you can set COMMIT_SHA, which will be appended to the image name we
 build as a debugging convenience.
 
 ```go
 func Test_Custom_Build_With_Docker(t *testing.T) {
-  binary := os.Getenv("VAULT_BINARY")
-  if binary == "" {
-    t.Skip("only running docker test when $VAULT_BINARY present")
-  }
-  opts := &docker.DockerClusterOptions{
-    ImageRepo: "hashicorp/vault",
-    // We're replacing the binary anyway, so we're not too particular about
-    // the docker image version tag.
-    ImageTag:    "latest",
-    VaultBinary: binary,
-    ClusterOptions: testcluster.ClusterOptions{
-      VaultNodeConfig: &testcluster.VaultNodeConfig{
-        LogLevel: "TRACE",
-      },
-    },
-  }
+  opts := docker.DefaultOptions(t)
   cluster := docker.NewTestDockerCluster(t, opts)
   defer cluster.Cleanup()
 }
 ```
 
 There are a variety of helpers in the `github.com/hashicorp/vault/sdk/helper/testcluster`
 package, e.g. these tests below will create a pair of 3-node clusters and link them using
-PR or DR replication respectively, and fail if the replication state doesn't become healthy.
-These depend on having a vault-enterprise binary locally and the env var VAULT_BINARY set to
-point to it, as well as having VAULT_LICENSE_CI set.
+PR or DR replication respectively, and fail if the replication state doesn't become healthy
+before the passed context expires.
+
+Again, as written, these depend on having a Vault Enterprise binary locally and the env
+var VAULT_BINARY set to point to it, as well as having VAULT_LICENSE_CI set.
 
 ```go
 func TestStandardPerfReplication_Docker(t *testing.T) {
   opts := docker.DefaultOptions(t)
-  opts.ImageRepo = "hashicorp/vault-enterprise"
   r, err := docker.NewReplicationSetDocker(t, opts)
   if err != nil {
       t.Fatal(err)
   }
   defer r.Cleanup()
 
-  err = r.StandardPerfReplication(context.Background())
+  ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
```

Comment:
nit: is a minute long enough to not be too flaky in our experience? People will be able to figure that out of course but if these examples are reasonable starting points to copy directly including sensible timing info that seems more useful.

I don't know that it isn't reasonable as it is, but I've seen a few of the more complex examples have timeouts of multiple minutes to be more resilient in slow CI.